app/collections/tracks.js

```diff
@@ -1,24 +1,49 @@
 import Backbone from 'backbone';
 import Track from '../models/track';
+import cozysdk from 'cozysdk-client';
 
 
 const Tracks = Backbone.Collection.extend({
     model: Track,
+
+    initialize: function(models, options) {
+        this.on('add', this.onAdd, this);
+        if (options) {
+            this.type = options.type;
+        }
+        this._attributes = {}
+    },
+
+    setAttr: function(prop, value) {
+        this._attributes[prop] = value;
+        this.trigger('change:' + prop, value);
+    },
+
+    getAttr: function(prop) {
+        return this._attributes[prop];
+    },
+
+    onAdd: function(track) {
+        if (!track.get('_id')) {
+            track.save();
+        }
+    },
     
     comparator: function (collection) {
         return collection.get('metas').title;
```

Comment:
Here you can't pass a string for the comparator as the value is nested in an object, but use the `sortBy` sort syntax with ES6: `comparator (model) { return model.get('metas').title }`